src/main/scala/model/genome/Kinds.scala

```diff
@@ -103,6 +103,15 @@ object GenesUtils {
    */
   def assignRandomDominance(): Unit =
     Genes.values.foreach(gk => setAlleleDominance(List(gk.base, gk.mutated)(Random.nextInt(2))))
+
+  /**
+   * @param gene  the Gene of which the dominant Allele is needed
+   * @return      the dominant Allele associated to the Gene specified
+   */
+  def getDominantAllele(gene: Gene): Allele = {
```

Comment:
A livello di `Kinds` non si dovrebbero usare i Geni ed Alleli "schietti", che sono dedicati agli specifici Bunny. Ll'intestazione del metodo dovrebbe essere `def getDominantAllele(geneKind: GeneKind): AlleleKind`, infatti avendo il GeneKind possiamo controllare se per esso è già stata settata la dominanza (se non lo è possiamo tirare un'eccezione), nel caso sia settata sempre a partire dal GeneKind restituiamo l'AlleleKind `base` oppure quello` mutated` (in base a quale dei due ha isDominant a true).
Comunque dato che per ora non l'abbiamo usata e magari non ci serve neppure non è fondamentale.